media/css/sandstone/buttons.less

```diff
@@ -333,6 +333,15 @@ ul.download-list {
     display: block !important;
 }
 
+// Add another rule for old browsers where :not isn't supported. If this is
+// mixed with the rule above, the entire rule will be invalid on those legacy
+// environments and some tests will fail.
+.osx:not(.pre-mavericks) .download-button .os_osx,
+.osx.pre-mavericks .download-button:not(.mavericks-required) .os_osx {
+    display: block !important;
+}
```

Comment:
Where's Firefox 3 from? As per MDN, [`:not`](https://developer.mozilla.org/en-US/docs/Web/CSS/:not) is supported since Firefox 1.